### DISCUSSION TITLE
installing pimcore datahub [ERROR]

### DISCUSSION THREAD

**Discussion by Yuki1602** [pimcore/pimcore#10811] [Q&A / Support] [answered]
Attempted to load class "PimcoreLocationAwareConfigDao" from namespace "Pimcore\Model\Dao".
Did you forget a "use" statement for another namespace?

this is the error i get when i try to open  my admin backend after  i run composer require command for datahub

**Comment by dvesh3** [comment, marked as answer]
You need to use Pimcore version >= 10.2 to use latest version of datahub bundle.

**Reply by blankse** [reply]
@dvesh3 Why not bump version in composer.json when this is the requirement?

**Reply by dvesh3** [reply]
yes, we need to..

**Reply by Yuki1602** [reply]
the datahub is being installed in the system after changing to 10.2
![image](https://user-images.githubusercontent.com/61821237/141281065-cda29557-028b-449e-b7d1-13a3ba9d65ab.png)
it shows its enabled
when i click on to install it in pimcore backend it shows success but doesnt install it and shows the install icon again after reload

**Reply by dvesh3** [reply]
yes, that's a bug. Already created a PR https://github.com/pimcore/data-hub/pull/437 to fix the install issue. When the fix is confirmed, i'll tag 1.1.1 release soon.
